packages/lxml/meta.yaml

```diff
@@ -0,0 +1,14 @@
+package:
+  name: lxml
+  version: 4.4.1
+source:
+  sha256: c81cb40bff373ab7a7446d6bbca0190bccc5be3448b47b51d729e37799bb5692
+  url: https://files.pythonhosted.org/packages/c4/43/3f1e7d742e2a7925be180b6af5e0f67d38de2f37560365ac1a0b9a04c015/lxml-4.4.1.tar.gz
+build:
+  cflags: -I../../../../libxml/libxml2-2.9.9/include -I../../../../libxslt/libxslt-1.1.33/ -Wno-implicit-function-declaration
+  ldflags: -L../../../../libxml/libxml2-2.9.9/.libs/ -L../../../../libxslt/libxslt-1.1.33/libxslt/.libs/
+test:
+  imports:
+  - lxml
+  - lxml.etree
+  - lxml.objectify
```

Comment:
If I remove these two imports, the test succeeds. I think this is pretty much what we expect, right?

Reply:
Hi @fallaciousreasoning , sorry if I'm breaking GH rules (I'm fairly new in GH), but I'm looking for lxml in pyodide, need it for XML processing from webbrowser.
Do you already have compiled lxml wasm that I can test?
As far as I understand, lxml.etree and lxml.objectify removal resolves the error.

Reply:
So kind of. There's an issue where it doesn't work until you import it a second time (which is why those tests fail).

I think I (maybe) have a build somewhere, but I don't know if it will Just Work(TM) for you. I'm also travelling for a while, so it might be easier (and faster) if you compile my branch yourself.

Reply:
@fallaciousreasoning , thanks for reply. 
I copied your branch with intention to build, but did not pay attenttion to prerequistes. My dev. is in windows. To build the lxml for pyodid I'll need time to install Debian and prepare the env.
If you have the lxml build handy, I'll appreciate if you share.
Regards, Vasiliy

Reply:
If you're in a hurry, you should try building it with Docker: It will handle all the dependencies for you (it's going to be a while before I'm back in front of a laptop).